src/models/room.ts

```diff
@@ -3292,6 +3295,24 @@ export class Room extends ReadReceipt<RoomEmittedEvents, RoomEventHandlerMap> {
         }
         event.applyVisibilityEvent(visibilityChange);
     }
+
+    /**
+     * Find when a client has gained thread capabilities by inspecting the oldest
+     * threaded receipt
+     * @returns the timestamp of the oldest threaded receipt
+     */
+    public getOldestThreadedReceiptTs(): number {
+        return this.oldestThreadedReceiptTs;
+    }
+
+    /**
+     * Returns the most receipt unthreaded receipt for a given user
+     * @param userId - the MxID of the User
+     * @returns an unthreaded Receipt
```

Comment:
I've added a comment. But it's because some folks can use private read receipts, or disable sending receipts altogether

Reply:
:+1: thanks. In case it wasn't obvious, the question was somewhat rhetorical, and what I should have said was "please *document* what it means if this returns undefined" :smile: